CHANGELOG-1.8.md

```diff
@@ -12,6 +12,7 @@
 
 ### Constructors
 
+- Extract methods `schedule*` of `Pim\Component\Catalog\Completeness\CompletenessGeneratorInterface` into a `Pim\Component\Catalog\Completeness\CompletenessRemoverInterface`. Methods `schedule`, `scheduleForFamily` and `scheduleForChannelAndLocale` have been renamed respectively `removeForProduct`, `removeForFamily` and `removeForChannelAndLocale`.
```

Comment:
I'm not sure about the remove. I think **erase** or **delete** is somewhat more clear. Remove may imply that is has been removed from the product but can still exist outside of it. **drop** would work too.